modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/AllowUsersToEditSchemaProperties.testcase

```diff
@@ -196,6 +277,53 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test PropertiesRemovedFromListAreEnabledInPropertiesSidebar {
+		task ("And Given 'name', 'External Reference Code', 'Create Date', 'ID' fields placed in Properties") {
+			APIBuilderUI.goToEditRelatedEntryInEditApplication(
+				entity = "Schemas",
+				name = "University",
+				tabEntity = "Properties",
+				title = "My-app");
+
+			Click(
+				key_text = "name",
+				locator1 = "APIBuilder#PROPERTY_CONTAINER");
+
+			Click(
+				key_text = "External Reference Code",
+				locator1 = "APIBuilder#PROPERTY_CONTAINER");
+
+			Click(
+				key_text = "Create Date",
+				locator1 = "APIBuilder#PROPERTY_CONTAINER");
+
+			Click(
+				key_text = "ID",
+				locator1 = "APIBuilder#PROPERTY_CONTAINER");
+		}
```

Comment:
for the Click to add 'name, ERC, Create Date, and ID', I think you can try to use 'for loop'
```
for (var fieldName : list ${fieldNameList}) {
				Click(
					key_text = ${fieldName},
					locator1 = "APIBuilder#PROPERTY_CONTAINER");
			}
```